src/brevitas/nn/quant_bn.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         super(BatchNorm1dToQuantScaleBias, self).__init__(
             num_features,
             bias=True,
-            runtime_shape=(1, -1, 1),
+            runtime_shape=(1, -1),
```

Comment:
I think this has to stay as it was before, i.e. (1, -1, 1).
The reason is that BatchNorm1d is supposed to handle both 2D and 3D inputs. In case you need specifically to use a 2D input, you could always specify the runtime shape when instantiating the class.

Reply:
Sure thing. I haven't really made any serious test cases before, so this will take some time (since I would have to read all the prerequisites), but I could definitely make some drafts.

Reply:
I have also made the changes that you requested in my dev branch, but how do I update the pull request? Do I start another pull request? Thank you

Reply:
Every change, including commits, that you do in your dev branch will be automatically added to this PR, so when you have the tests, just add them to the same dev branch